tests/test_cli/test_cli_exceptions.py

```diff
@@ -44,7 +44,13 @@ def test_no_fact_module(self):
     def test_no_fact_cls(self):
         self.assert_cli_exception(
             ["my-server.net", "fact", "server.NotAFact"],
-            "No such attribute in module server: NotAFact",
+            (
+                "No such attribute in module server: NotAFact\n"
+                "Available facts in module are: User, Home, Path, TmpDir, Hostname, Kernel, "
```

Comment:
Each update will implies an updtate of the test, maybe use a regex slould be better?